walletkit/android/eip5792.mdx

```diff
@@ -12,41 +12,139 @@ Applications can specify that these onchain calls be executed taking advantage o
 
 ## Usage
 
-### wallet_getCapabilities Example
-
-The `wallet_getCapabilities` method is used to request information about the capabilities supported by a wallet. In accordance with EIP-5792, here’s how it should be implemented:
-
-### Request
-```json 
-{
-  "id": 1,
-  "jsonrpc": "2.0",
-  "method": "wallet_getCapabilities",
-  "params": ["0xd46e8dd67c5d32be8058bb8eb970870f07244567", ["0x2105", "0x14A34"]]
-}
-```
-
-### Response
-The wallet should return a response compliant with EIP-5792, where capabilities are organized by chain ID:
-
-```json
-{
-  "id": 1,
-  "jsonrpc": "2.0",
-  "result": {
-    "0x2105": {
-      "atomic": {
-        "status": "supported"
+<AccordionGroup>
+  <Accordion title="wallet_getCapabilities" defaultOpen>
+  ### Example
+    The `wallet_getCapabilities` method is used to request information about what capabilities a wallet supports. Following EIP-5792, here's how it should be implemented:
+
+    #### Request
+    ```json
+    {
+      "id": 1,
+      "jsonrpc": "2.0",
+      "method": "wallet_getCapabilities",
+      "params": ["0xd46e8dd67c5d32be8058bb8eb970870f07244567", ["0x2105", "0x14A34"]]
+    }
+    ```
+
+    #### Response
+    The wallet should return a response following EIP-5792, where capabilities are organized by chain ID:
+
+    <Note>
+    For non-atomic execution, include all transactions in the receipts array, even those that were included on-chain but eventually reverted.
```

Comment:
Not sure I understand this note, could you clarify? 
If they want to force non-atomic execution they should sequentially call sentTransaction instead as wallet is not guaranteed to execute non-atomically even if atomicRequired is false